workflow.rst

```diff
@@ -378,11 +378,34 @@ order:
     * ``workflow.[workflow name].announce``
     * ``workflow.[workflow name].announce.[transition name]``
 
+    You can avoid triggering those events by using the context::
+
+        $workflow->apply($subject, $transitionName, [Workflow::DISABLE_ANNOUNCE_EVENT => true]);
+
+    .. versionadded:: 5.1
+
+        The ``Workflow::DISABLE_ANNOUNCE_EVENT`` constant was introduced in Symfony 5.1.
+
+    .. versionadded:: 5.2
+
+        In Symfony 5.2, the context is accessible in all events::
+
+            // $context must be an array
+            $context = ['context'];
```

Comment:
```suggestion
            $context = ['context_key' => 'context_value'];
```
and swap below, wdyt? the context is aimed to be used mainly with key->val usage

Reply:
why not !